app/javascript/app/components/sdg-card/sdg-card-component.jsx

```diff
@@ -2,23 +2,25 @@ import React, { PureComponent } from 'react';
 import PropTypes from 'prop-types';
 import cx from 'classnames';
 
-import sdg1 from 'assets/sdg-icons/icon_1';
-import sdg2 from 'assets/sdg-icons/icon_2';
-import sdg3 from 'assets/sdg-icons/icon_3';
-import sdg4 from 'assets/sdg-icons/icon_4';
-import sdg5 from 'assets/sdg-icons/icon_5';
-import sdg6 from 'assets/sdg-icons/icon_6';
-import sdg7 from 'assets/sdg-icons/icon_7';
-import sdg8 from 'assets/sdg-icons/icon_8';
-import sdg9 from 'assets/sdg-icons/icon_9';
-import sdg10 from 'assets/sdg-icons/icon_10';
-import sdg11 from 'assets/sdg-icons/icon_11';
-import sdg12 from 'assets/sdg-icons/icon_12';
-import sdg13 from 'assets/sdg-icons/icon_13';
-import sdg14 from 'assets/sdg-icons/icon_14';
-import sdg15 from 'assets/sdg-icons/icon_15';
-import sdg16 from 'assets/sdg-icons/icon_16';
-import sdg17 from 'assets/sdg-icons/icon_17';
+import Icon from 'components/icon';
```

Comment:
Why don't use this https://github.com/Vizzuality/climate-watch/commit/cc48005927e83801af56f88c246866b92b8d5ca3#diff-c73b648ba0a802388cc26a5734d76122R14 ?
It imports the images dynamically what makes less annoying, isn't it?